spectre_analyses/celery_config.py

```diff
@@ -0,0 +1,2 @@
+broker_url = 'amqp://guest:guest@rabbitmq'
+result_backend = 'db+postgresql://guest:guest@postgresql/celery'
```

Comment:
Again, wrap both with `import os; os.environ.get(KEY, DEFAULT)`.

Reply:
No! This is config file as specified in Celery documentation. If we are to use environment variables, we should remove this module at all.